src/distributions/common_interface.jl

```diff
@@ -1,15 +1,16 @@
 # Currently supported distributions
 const DISTS = [
     Normal,
```

Comment:
I suggest just organising the distributions in alphabetical order

Reply:
gonna do in the next one